com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -2515,18 +2536,58 @@ private void SetDefaultCamera (FbxScene fbxScene)
         /// </summary>
         /// <returns>Unique name</returns>
         /// <param name="name">Name</param>
-        private string GetUniqueName(string name)
+        /// <param name="nameToIndexMap">The dictionary to use to map name to # of occurences</param>
+        private string GetUniqueName(string name, ref Dictionary<string, int> nameToIndexMap)
```

Comment:
doesn't need to be a ref I don't think; you're not assigning to nameToIndexMap

Reply:
(if we were building lots and lots of tiny FbxExporters we might want to set the Dictionary to null in the constructor, and in GetUniqueName we'd allocate it if it was stil null. But we aren't)